dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -0,0 +1,903 @@
+<template>
+  <Card class="col-12 page-wrapper-card">
+    <template #title>New Dataset - LkSG </template>
+    <template #content>
+      <div class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
```

Comment:
What about share of temporary workers? Are they no longer a part of the data model?